src/main/resources/templates/main.html

```diff
@@ -33,6 +33,9 @@
                       <li class="nav-item"><a class="nav-link" aria-current="page" href="/_renarde/backoffice/index"><i class="bi bi-database"></i>{m:main.backoffice}</a></li>
                     {/if}
                     <li class="nav-item"><a class="nav-link" aria-current="page" href="{uri:RenardeSecurityController.logout()}">{m:main.logout}</a></li>
+                    {#if inject:user.tenantId && inject:user.tenantId is 'apple'}
+                      <li class="nav-item"><a class="nav-link" aria-current="page" href="{uri:RenardeRevokeController.revokeApple()}" >{m:main.revoke}</a></li>
```

Comment:
I don't think this is the right thing to show/do.

The original requirement from Apple is that it be possible to delete an account, and that deleting an account should revoke the tokens.

So I don't think this is the functionality we should expose, as it does not delete the local account (the `User` instance in the DB).

There should be an option to delete your account, probably with an intermediate page with a  "Are you really sure?" button, and that should delete the account, and revoke the tokens, no?

Reply:
That's right, we must also delete the user on our db, not only the apple access. 
Don't you think we need an User detail page to place this new button (and not in the top menu) ? 
For instance, a link on the user name in the top bar that open a page which display user info plus a revoke/delete account button.
Also, I'll need a custom endpoint on this Todo project to handle de DB deletion of the user. I can do that.

Reply:
Oh, sorry, I missed this.

Yes, you're right, it makes sense to add an account page, and put the `delete` button there.